### ISSUE TITLE
Proposal: LinkedIn Group for BackdropCMS

### ISSUE THREAD

**Issue by dougvann** [backdrop-ops/backdropcms.org#131]
Hey everyone.. Remember me!!
I've been enjoying [Issue# 122] (https://github.com/backdrop-ops/backdropcms.org/issues/122) Monkey's Marketing thread since it started 12 days ago. I had a response all drafted immediately and accidentally wound up rebooting before sending it. :-(
Anyway... here I am... I've been invisible but I continue to fight the good fight and sell the value propositions of BackdropCMS. See [my latest blog post] (http://dougvann.com/blog/drupal-8-launches-today-and-here-i-sit-thinking-about-backdropcms) for yet more fun! 

Rather than trying to reconstruct that lost post, here's a fresh one.

One big issue any group runs into is fragmented conversations. Slack / IRC / SkypeChat / GoogleHangOuts, etc. We've all been there.
When I propose that we create a LinkedInGroup, I do so not as a means to fragment our current conversations, but as a means to capture some attention over there. Here's how I see it happening.
 1.  Someone starts the group and names it BackdropCMS
 2.  Among the fields required for a new LinkedIn Group are Summary, Description & Website. This obviously gives us a chance to promote. Assuming we mention our roots, we will show up on searches for DRUPAL
 3.  While we don't want to move major conversations over there, BUT we could announce version releases, and other marketing type data. We might be surprised at what LinkedIn users may have to say in there. 
 4.  If some conversation become actionable, we can create a new issue and have an internal conversation here.

Risks:
 1.  Takes some effort, maybe not a lot, but some. We must respond to activity in the group including removing the occasional spam
 2.  Could become a battleground for naysayers where we continuously argue the viability of BackdropCMS although I get less of that at camps and user-groups than I used to.
 3.  On the other hand, it could be a wasteland with our voices echoing off the walls. We could mitigate that by purposely moving some conversations over there if we felt it was warranted. Either way, we need to SEED the group if we expect to attract any conversation out there.

What do you think?
Is it worth it? Would we have a clear vision? Who wants to own the group? We could all participate if we choose to create it.

Thoughts?

**Comment by klonos** [comment]
I'm sorry, but I do not have the time/energy/interest for this. I do not object to the basic idea of creating a "minimal" profile for marketing/SEO/networking purposes at all though. Just announcements to begin with wouldn't hurt.

**Comment by docwilmot** [comment]
I think a LinkedIn presence is a good idea although I suspect it will end up being mostly static. I have little experience there though, so I'd recommend @dougvann be left in charge of this: build it and see what happens.

**Comment by klonos** [comment]
What the doc said :+1:

**Comment by sutibun** [comment]
> Hey everyone.. Remember me!!

@dougvann you left us and now you want back.... not until you bring monkey a banana as an apology :p

Just hasn't been the same without you.... whoever you are, sir ^_-

Anyway, monkey no have anything against creating a LinkedIn group thingy really. Just hesitant of how effective it'll be. But worth a try.

What we can do if no one has time is maybe automatically post Bd blog posts to the LI group page using what this guys mentions FB over here https://github.com/backdrop-ops/backdropcms.org/issues/133

Monkey nominates @dougvann to manage the LI group. hehehe

**Comment by thejimbirch** [comment]
I can set this up and manage if no one objects.

**Comment by sutibun** [comment]
@thejimbirch Hired! Oh wait... monkey has no real power ^^;

**Comment by thejimbirch** [comment]
LinkedIn group created:

https://www.linkedin.com/groups/8457093

If you want to be admin, let me know.

**Comment by dougvann** [comment]
Excellent work, Jim.
I see that the group requires approval to join. Was that intentional? I was thinking this would be public.
And I'd love to help out as an admin.
THNX!

**Comment by thejimbirch** [comment]
@dougvann I believe all groups now need approval to join.  I could not find settings to change this.  I added you as an admin, thanks!

**Comment by dougvann** [comment]
Ahh. Right you are. It's been that way for a while. I don't know if I was thinking of the deep past or some other network's groups. Aye yi yi... Hard to keep all that straight.
THNX!

**Comment by thejimbirch** [comment]
Could we get a link to this on the site?  Maybe a Tweet or two?

https://www.linkedin.com/groups/8457093

**Comment by jenlampton** [comment]
Sorry I missed this thread :/  The Backdrop CMS LinkedIn group is here: https://www.linkedin.com/groups/6930143

**Comment by jenlampton** [comment]
@thejimbirch let's switch the newer group for the older one, since it has a few more members (and the official project name, complete with "CMS".  Could you or @dougvann put a post in the newer group pointing members to the older one?  I tried to invite everyone I could see in there too, but apparently I'm not connected to all of you yet! ;)

**Comment by thejimbirch** [comment]
Hi @jenlampton 

Sent a message to the members alerting them of your group, and deleted the group I started.

Tweet sent out, RT's welcome:

https://twitter.com/backdropcms/status/713199666460889088

Thanks,
Jim

**Comment by jenlampton** [comment]
woohoo!  ty @thejimbirch :)
